examples/test-dev-network.sh

```diff
@@ -29,3 +29,20 @@ for i in `seq $NUM_ORGS`; do
         -c '{\"Args\":[\"queryTraintuples\"]}'"
     echo '-----------'
 done
+
```

Comment:
Is the chaincode already approved/commited at the point in time when the chaincode is instantiated (i.e. the chaincode pod is running and Ready?)

If so, maybe we don't need this file at all anymore? In other words, the chaincode pods being up-and-running guarantees that the chaincode is invokable.

Reply:
Chaincode is running and ready by itself and only need package chaincode CCID.
Approval and commit steps are for the channel to allow transaction from that chaincode.